### PR TITLE
poc for python e2e scripts

### PR DESCRIPTION
**Pull request by dorin100**
constants.py - file where to store constant value variables
utils.py - file where storing reusable functions
transaction_basic.py - basic script for poc

### CODE REVIEW

File: python_scripts/e2e_scenarios/utils.py [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+        print(f"Folder does not exists - {location_offline_tx_folder}")
+
+
+def set_node_socket_path_env_var():
+    os.environ['CARDANO_NODE_SOCKET_PATH'] = NODE_SOCKET_PATH
+
+
+def create_payment_key_pair(location, key_name):
+    try:
+        cmd = "cardano-cli shelley address key-gen --verification-key-file " \
```
Comment (ArturWieczorek) [inline python_scripts/e2e_scenarios/utils.py:27]: I would put each CLI switch/option for this command on separate line - just like in `build_payment_address` below

File: python_scripts/e2e_scenarios/utils.py [hunk 2]
```diff
@@ -0,0 +1,319 @@
+import re
+import shutil
+import subprocess
+import os
+from time import sleep
+
+from e2e_scenarios.constants import TESTNET_MAGIC, PROTOCOL_PARAMS, NODE_SOCKET_PATH, USER1_ADDRESS
```
Comment (ArturWieczorek) [inline python_scripts/e2e_scenarios/utils.py:7]: I think it's OK to list them all here for now but if the list of constants will grow then maybe we will need to use import all *

Comment (dorin100) [reply python_scripts/e2e_scenarios/utils.py:7]: Ideally, we should import only what we use/need but we can discuss it once it becomes inconvenient :)

File: python_scripts/e2e_scenarios/utils.py (hunk 1 above)
Comment (dorin100) [reply python_scripts/e2e_scenarios/utils.py:27]: yeap. will be part of the next commit/pr

**Pull request merged**